### PR TITLE
added flt dns commands

### LINKED ISSUE #167: add DNS entries from flt
- read IPs file and re-create DNS entries
- support --grep to limit
- only add / update if not correct

We also need to figure out which script is deleting DNS entries

### PR DESCRIPTION
**Pull request by bartr**
## Purpose of PR

- added flt dns commands

## Type of PR

- [ ] Documentation changes
- [x] Code changes
- [x] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Issues Closed or Referenced

- Closes #167

### CODE REVIEW

File: src/kic/cover.out [hunk 1]
```diff
@@ -8,23 +8,6 @@ kic/boa/cfmt/cfmt.go:59.2,61.47 2 0
 kic/boa/cfmt/cfmt.go:65.2,65.12 1 0
```
Comment (wbreza) [inline src/kic/cover.out:8]: Do we even want these coverage files in source control?  Seems like maybe a build artifact would be more appropriate.

File: vm/setup/akdc-install.sh [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 # this script installs most of the components
 
+# this fails sometimes so we cache
+cp "$HOME/cli/vm/setup google-apt-key.gpg" .
```
Comment (wbreza) [inline vm/setup/akdc-install.sh:6]: What is this key used for?

**Review by wbreza — APPROVED** [review]
Looks good - left a couple comments around usage of gpg key.

File: src/kic/cover.out (hunk 1 above)
Comment (bartr) [reply src/kic/cover.out:8]: having them in source allows us to have a coverage map linked from the repo - see the link in readme

File: vm/setup/akdc-install.sh (hunk 2 above)
Comment (bartr) [reply vm/setup/akdc-install.sh]: I removed it - I thought the curl command on that file was failing but it's actually the apt-get update that's failing to pull something.

**Pull request merged**